src/components/Button/Button.less

```diff
@@ -17,6 +18,10 @@
 	line-height: @Button-size-lineHeight;
 	padding: 0 @Button-size-padding;
 
+	&[disabled] {
```

Comment:
Shouldn't this be

```
&-is-disabled {}
```

Per [this class in the code](https://github.com/appnexus/lucid/blob/master/src/components/Button/Button.jsx#L111)